### DISCUSSION TITLE
propEq

### DISCUSSION THREAD

**Discussion by viclafouch** [remeda/remeda#849] [Suggest a Function]
I know it was discussed in https://github.com/remeda/remeda/issues/37 but I think `propEq` was one of the utility I used a lot when I used Ramda. 

Doing 

```
// Not this
const post = posts.find((item) => {
  return item.subject === 'toto'
})

// But this
const post = posts.find(R.propEq('subject', 'toto'))
```

Was very nice for Array.`find | filter | some | every..` etc...

Is there an equivalent in Remeda, or, if I want to build my own `propEq` in Typescript, do you have an idea ? 

Thanks a lot !

**Comment by cjquines** [comment]
`propEq` could probably fit in the `hasProp` or `hasPropSatisfying` requested in #461 ?

**Comment by viclafouch** [comment]
Yes totally! What do you think about these 2 new features ? 

Looks like the [PR](https://github.com/remeda/remeda/pull/515) has never been rebased 😭

**Reply by cjquines** [reply]
:shrug: i created #853. i would accept a pr for it
